gateway/transport/plugins/audit/audit.go

```diff
@@ -180,13 +189,56 @@ func (p *auditPlugin) OnDisconnect(pctx plugintypes.Context, errMsg error) error
 	return nil
 }
 
+func parseSessionToFile(s *types.Session, opts sessionParseOption) []byte {
+	output := []byte{}
+	for _, eventList := range s.EventStream {
+		event := eventList.(types.SessionEventStream)
+		eventType, _ := event[1].(string)
+		eventData, _ := base64.StdEncoding.DecodeString(event[2].(string))
+
+		if !slices.Contains(opts.events, eventType) {
+			continue
+		}
+
+		switch eventType {
+		case "i":
+			output = append(output, eventData...)
+		case "o", "e":
+			output = append(output, eventData...)
+		}
+		if opts.withLineBreak {
+			output = append(output, '\n')
+		}
+	}
+	return output
+}
+
 func (p *auditPlugin) closeSession(pctx plugintypes.Context, errMsg error) {
 	log.With("sid", pctx.SID).Infof("closing session, reason=%v", errMsg)
 	go func() {
 		if err := p.writeOnClose(pctx, errMsg); err != nil {
 			log.Warnf("session=%v - failed closing session: %v", pctx.SID, err)
 			return
 		}
+
+		session, err := sessionstorage.FindOne(pctx, pctx.SID)
+		if err != nil {
+			log.Error("failed obtaining session, err=%v", err)
+		}
+
+		events := []string{"o", "e"}
+		payload := parseSessionToFile(session, sessionParseOption{withLineBreak: true, events: events})
+
+		payloadLength := len(payload)
+		if payloadLength > 5000 {
+			payloadLength = 5000
+		}
+
+		issueInfo := jira.AddSessionExecutedIssueTemplate{
+			Payload: string(payload[0:payloadLength]),
+		}
+		jira.AddSessionExecutedJiraIssue(pctx.OrgID, session.JiraIssue, issueInfo)
```

Comment:
I recommend logging the error.
Add a _ to indicate that you won't be dealing with the error.